Formula/got.rb

```diff
@@ -1,10 +1,15 @@
 class Got < Formula
   desc "Version control system"
-  homepage "https://gameoftrees.org"
-  url "https://gameoftrees.org/releases/portable/got-portable-0.75.tar.gz"
-  sha256 "b35324893c9aefbd6bf49602a6c9bb9d6e20670aff3c9734fefead0205bb9cb2"
+  homepage "https://gameoftrees.org/"
+  url "https://gameoftrees.org/releases/portable/got-portable-0.75.1.tar.gz"
+  sha256 "1a511707cf5f64f459030bd45077322821f7263e403af3121c5a1db1529ea524"
   license "ISC"
 
+  livecheck do
+    url "https://gameoftrees.org/releases/portable/"
+    regex(/href=.*?got[._-]portable[._-]v?(\d+(?:\.\d+)+)\.t/i)
```

Comment:
```suggestion
    regex(/href=.*?got-portable[._-]v?(\d+(?:\.\d+)+)\.t/i)
```

For the first part of the filename before the version (typically the software name), I usually just leave hyphens as-is, like `got-portable[._-]` instead of `got[._-]portable[._-]`. The former is more flexible but kind of leads us down a rabbit hole of using `[._-]` everywhere and produces lengthier regexes overall. We usually don't benefit from the more flexible/verbose approach when it comes to the software name part of filenames, so I only introduce the more flexible approach when necessary/appropriate.

Reply:
That seems pretty arbitrary, but ok

Reply:
It's just about consistency in this instance. This is how we've approached this style of regex for the past couple years, so almost all regexes with a multipart name (e.g., `abc-def` instead of `abc`) follow this pattern. Regardless of whether we maintain this approach or switch to the more flexible/verbose approach in the long run, there's something to be said for keeping them consistent.